### PR TITLE
feat: show milliseconds in timestamp in logs views

### PR DESCRIPTION
**Pull request by YounixM**
Show milliseconds in timestamp in logs views

Format
YYYY-MM-DD HH:mm:ss.SSS

Screenshot 2024-05-01 at 12 15 26 PM

### CODE REVIEW

File: frontend/src/components/Logs/TableView/useTableView.tsx [hunk 1]
```diff
@@ -67,6 +67,8 @@ export const useTableView = (props: UseTableViewProps): UseTableViewResult => {
 			return [...fieldColumns];
 		}
 
+		console.log('fieldColumns', fieldColumns);
```
Comment (SagarRajput-7) [inline frontend/src/components/Logs/TableView/useTableView.tsx:70]: remove console log

**Review by SagarRajput-7 — DISMISSED** [review]
lgtm

File: frontend/src/components/Logs/TableView/useTableView.tsx (hunk 1 above)
Comment (Vikrant2520) [reply frontend/src/components/Logs/TableView/useTableView.tsx]: done

**Review by Vikrant2520 — APPROVED** [review]

**Review by nityanandagohain — APPROVED** [review]
lgtm!

**Pull request merged**